frontend/src/components/abfragevarianten/SachbearbeitungComponent.vue

```diff
@@ -35,9 +37,12 @@
             :rules="[fieldValidationRules.pflichtfeld]"
             required
           />
-        </v-col>
+        -->
       </v-row>
-      <sobon-berechnung v-model="abfragevarianteSachbearbeitung.sobonBerechnung"></sobon-berechnung>
+      <sobon-berechnung
+        v-if="isBauleitplanverfahrenOrWeiteresVerfahren"
+        v-model="abfragevarianteSachbearbeitung.sobonBerechnung"
+      ></sobon-berechnung>
```

Comment:
Ein öffnender und schließender HTML-Tag `<sobon-berechnung>` ist nicht erforderlich, da innerhalb der Komponenten keine weitern Komponenten integriert werden.

Das schließende Tag in den öffnenden Tag integrieren:
```html
<sobon-berechnung
        v-if="isBauleitplanverfahrenOrWeiteresVerfahren"
        v-model="abfragevarianteSachbearbeitung.sobonBerechnung"
      />
```